src/main/java/com/simpletalk/enterprise/SimpleTalkController.java

```diff
@@ -24,15 +24,23 @@ public class SimpleTalkController {
 
     /**
      * Handle the / endpoint
+     *
      * @return
      */
     @RequestMapping("/")
     public String index(Model model) {
+        Post post = new Post();
+        post.setMessage("new random message");
+        post.setPostID(37);
+        post.setLikes(50);
+        model.addAttribute(post);
         return "startPage";
     }
 
     @RequestMapping("/explore")
-    public String explore() { return "explorePage"; }
```

Comment:
made the spacing correct. so its more readable

Reply:
Use Control-Alt-S in IntelliJ IDEA to perform formatting automatically.